### DISCUSSION TITLE
I am keeping redirected without a specific reason

### DISCUSSION THREAD

**Discussion by HosMercury** [tokio-rs/axum#2607] [Q&A] [answered]
### Summary

this is my whole router , disable any middleware

```rs
    let app = dashboard::router()
        //.layer(middleware::from_fn(middlewares::auth_middlware))
        .merge(auth::router())
        .merge(oauth::router())
        .layer(session_layer)
        .with_state(state)
        .nest_service("/assets", ServeDir::new("assets"));
 ```
 
 I am kept redirected from `/` and `/t` to `/login` 
 
 without any reason
 
 here is my `dashboard.router` 
 
 ```rs
 pub fn router() -> Router<AppState> {
    Router::new().route("/", get(dashboard)).route("/t", get(t))
}

#[axum::debug_handler]
pub async fn dashboard() -> DashboardTemplate {
    println!("dashboard");

    DashboardTemplate {
        title: "Dashboard",
        messages: None,
        username: "Hos".to_string(),
    }
}

pub async fn t() -> DashboardTemplate {
    DashboardTemplate {
        title: "Dashboard",
        messages: None,
        username: "Hos".to_string(),
    }
}
```

I have no errors and I have disabled everything for testing ..
i have also revised all my redirects just in case 

got crazy !!

any idea?

### axum version

latest

**Comment by HosMercury** [comment, marked as answer]
it was a chrome issue

**Comment by ttys3** [comment]
actually you can avoid this issue.

remeber to not send  these http code to Chrome:

](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/301)
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/308

instead you should use:

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/302

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/307

**Reply by HosMercury** [reply]
Ty
